terraform/variables.tf

```diff
@@ -45,3 +45,8 @@ variable "memorystore" {
   type        = bool
   description = "If true, Online Boutique's in-cluster Redis cache will be replaced with a Google Cloud Memorystore Redis cache"
 }
+
+variable "redis-enterprise" {
```

Comment:
```suggestion
variable "redis_enterprise" {
```

Reply:
Resolved in https://github.com/gmflau/microservices-demo/commit/94ee7736dae7a841face746f3dfc0c635c260150.